content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -28,9 +28,11 @@ a building block in other ecosystem and/or installer tool with a larger
 scope.
 
 You can install _kubeadm_ very easily on operating systems that support
-installing deb or rpm packages. The responsible SIG for kubeadm,
-[SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle), provides these packages pre-built for you,
-but you may also build them from source for other OSes.
+installing deb or rpm packages. The responsible
+{< glossary_tooltip text="SIG" term_id="sig" >}} for kubeadm,
+[SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle),
+provides these packages pre-built for you, but you may also build them
+from source for other operating systems.
```

Comment:
this is no longer true.

i would leave it to the following, without the extra details:

> You can install _kubeadm_ very easily on operating systems that support
installing DEB or PRM packages.